src/MSALWrapper/AuthFlow/Broker.cs

```diff
@@ -86,8 +83,8 @@ private enum GetAncestorFlags
         public async Task<AuthFlowResult> GetTokenAsync()
         {
             IAccount account = await this.pcaWrapper.TryToGetCachedAccountAsync(this.preferredDomain)
-                ?? Identity.Client.PublicClientApplication.OperatingSystemAccount;
-            this.logger.LogDebug($"Using cached account '{account.Username}'");
+                 ?? PublicClientApplication.OperatingSystemAccount;
+            this.logger.LogDebug($"Using cached account '{account?.Username}'");

```

Comment:
If the account is null, would it just print "Using cached account "?

Reply:
There are ticks `'` so it prints `Using cached account ''`

Reply:
This could be slightly improved to say <NO ACCOUNT> or something. What do you think?

Reply:
Looks good to me :) I just saw web and devicecode code is printing it at all if the account is empty.